src/test/components/ListOfPublishedProfiles.test.js

```diff
@@ -443,4 +443,46 @@ describe('ListOfPublishedProfiles', () => {
       );
     });
   });
+
+  describe('The uploaded recordings list should update on window focus', () => {
+    it('will update the stored profiles on window focus', async function() {
+      // Add 3 examples, all with the same name.
+      // mockDate('4 Jul 2020 15:00'); // Now is 4th of July, at 3pm local timezone.
+  
+      const exampleProfileData = {
+        profileToken: 'MACOSX',
+        jwtToken: null,
+        publishedDate: new Date('4 Jul 2020 13:00'),
+        name: 'Example Profile',
+        preset: null,
+        originHostname: 'https://mozilla.org',
+        meta: {
+          product: 'Firefox',
+          platform: 'Macintosh',
+          toolkit: 'cocoa',
+          misc: 'rv:62.0',
+          oscpu: 'Intel Mac OS X 10.12',
+        },
+        urlPath: '/public/MACOSX/marker-chart/',
+        publishedRange: { start: 2000, end: 40000 },
+      }
+  
+      await storeProfileData(exampleProfileData);
+      await storeProfileData(exampleProfileData);
+      await storeProfileData(exampleProfileData);
```

Comment:
You can't store 3 times the exact same profile date, because the property `profileToken` is a primary key: it identifies one profile and only one. This means that when storing the second one, it will overwrite the first one instead of adding it.

You can easily store 3 profiles with different profileToken values like this:
```js
await storeProfileData({
  ...exampleProfileData,
  profileToken: "PROFILE-1",
});
await storeProfileData({
  ...exampleProfileData,
  profileToken: "PROFILE-2",
});
await storeProfileData({
  ...exampleProfileData,
  profileToken: "PROFILE-3",
});
```